Dockerfile

```diff
@@ -20,6 +20,12 @@ RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so
   locale-gen en_US.UTF-8; \
   mkdir -p /var/lock/apache2 /var/run/apache2 /var/run/sshd /var/log/supervisor
 
+RUN apt-get update; \
+  echo "postfix postfix/mailname string drupal-mail" | debconf-set-selections ; \
+  echo "postfix postfix/main_mailer_type string 'Local only'" | debconf-set-selections ; \
+  apt-get install --assume-yes postfix mailutils; \
+  service postfix start
```

Comment:
Hello @richardkapolnai for this to work the service management needs to be handled by the supervisord.conf in the files folder. 
Please test it locally and make the necessary additions.

Reply:
Ok, done